k8s/configmap.yaml

```diff
@@ -2,18 +2,25 @@ apiVersion: v1
 kind: ConfigMap
 data:
   SCALYR_K8S_CLUSTER_NAME: <your-cluster-name>
-  SCALYR_K8S_DISABLE_API_SERVER: "true"
-  SCALYR_K8S_ALWAYS_USE_DOCKER: "true"
-  SCALYR_K8S_CACHE_EXPIRY_SECS: "600"
-  SCALYR_K8S_CACHE_START_FUZZ_SECS: "10"
-  SCALYR_K8S_CACHE_EXPIRY_FUZZ_SECS: "15"
-  SCALYR_COMPRESSION_TYPE: bz2
-  K8S_EVENTS_DISABLE: "true"
+  SCALYR_K8S_RATELIMIT_CLUSTER_NUM_AGENTS: "1000"
+  SCALYR_K8S_RATELIMIT_CLUSTER_RPS_INIT: "40"
+  SCALYR_K8S_RATELIMIT_CLUSTER_RPS_MIN: "8"
+  SCALYR_K8S_RATELIMIT_CLUSTER_RPS_MAX: "200"
+  SCALYR_K8S_RATELIMIT_CONSECUTIVE_INCREASE_THRESHOLD: "5"
```

Comment:
I think considering getting rid of lines 9-14 as they are identical to `configuration.py` defaults. WDYT? In case customer uses their own configmap (which was the case previously), the fewer the params, the less likely chance for mistakes.

Reply:
I think we should keep everything we might want them to tweak in the configmap.  That if the *do* need to change it, they only need to change a number rather than type in the name of the variable - that will have a larger potential for mistakes.

Reply:
I thought of this too but didn't think we'd need to modify these other params other than init/min/max.  I guess we can leave in just in case.